examples/life.js

```diff
@@ -0,0 +1,119 @@
+qKit.init({border:{width:8,color:'#824222'}});
+
+const suffusion	=192;
+
+var humor=new Array(2);
+humor[0]=new Array(9);
+humor[1]=new Array(9);
+
+//was dead
+humor[0][0]='#000000';
+humor[0][1]='#001840';
+humor[0][2]='#003050';
+humor[0][3]='#40FF60';//born
+humor[0][4]='#202840';
+humor[0][5]='#20283C';
+humor[0][6]='#1C2438';
+humor[0][7]='#182030';
+humor[0][8]='#101820';
+
+//was alive
+humor[1][0]='#602010';
+humor[1][1]='#702018';
+humor[1][2]='#D0FF00';//lives
+humor[1][3]='#F8FF60';//lives
+humor[1][4]='#803830';
+humor[1][5]='#44342C';
+humor[1][6]='#38302C';
+humor[1][7]='#302820';
+humor[1][8]='#281818';
+
+var habitat=new Array(2);
+habitat[0]=new Array(suffusion);
+habitat[1]=new Array(suffusion);
+for(var r=0;r<suffusion;r++)
+{
+	habitat[0][r]=new Array(suffusion);
+	habitat[1][r]=new Array(suffusion);
+}
+
+var phase=0;
+var auto=true;
+var scroll=true;
+
+function Random5050Habitat()
+{
+	phase=0;
+
+	for(var r=0;r<suffusion;r++)for(var c=0;c<suffusion;c++)
+	{
+		habitat[0][r][c]=(Math.floor(Math.random()*2)<<2)-1;

```

Comment:
Please use `qKit.util.randomInt(min, max)` or `qKit.util.randomDouble(min, max)`

Reply:
Will comply.  Same commentary as with the repeat.  It's a virtue to showcase qKit functionality but I'm not clear on what motivates having it since JavaScript coders would have greater facility reading and using the native equivalent.  Also note that the parameter-names are somewhat misleading.  In order to achieve an integer of either zero or one, with a 50/50 chance, the maximum value would be one but the call must be qKit.util.randomInt(0, 2).